ruleset/rector.php

```diff
@@ -2,19 +2,31 @@
 
 declare(strict_types=1);
 
-use Rector\CodeQuality\Rector\Class_\InlineConstructorDefaultToPropertyRector;
 use Rector\Config\RectorConfig;
-use Rector\Set\ValueObject\LevelSetList;
 use Rector\Set\ValueObject\SetList;
+use Rector\Symfony\Set\SymfonySetList;
 
-return static function (RectorConfig $rectorConfig): void {
-    $rectorConfig->paths([
-        dirname(__DIR__) . '/src',
-        dirname(__DIR__) . '/tests/PHPUnit',
+return RectorConfig::configure()
+    ->withPaths([
+        __DIR__ . '/src',
+        __DIR__ . '/tests/PHPUnit',
+    ])
+    ->withPHPStanConfigs([__DIR__ . '/ruleset/phpstan.neon'])
+    ->withPhpSets(php83: true)
```

Comment:
php82

Reply:
Agreed, however I didn't see any change related to 8.3 only